[CS2113-F13-1] Taste of Mom's

Aung-Phone-Naing opened the pull request:
Taste of Mom's helps students manage recipes, optimized for use via a Command Line Interface. This ensures that frequent tasks can be done faster by typing in commands.

liuziyang020319 commented:
Fetch: Clear and View

Jeraldchen commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,331 @@
+**Main components of the architecture**
+
+![image](./PlantUML/MainArchitecture.png)
+
```
Architecture diagram is clear and simple to understand.

Jeraldchen commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,331 @@
+The **API** of this component is specified in [`Parser.java`](https://github.com/AY2223S2-CS2113-F13-1/tp/blob/master/src/main/java/seedu/duke/parser/Parser.java)
+
+![image](./PlantUML/ParserComponent.png)
+
```
Remove the C and I symbols beside the Class and Interface boxes as it is not the module convention. Replace the private and public access modifiers with - and + symbols instead.

Jeraldchen commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,331 @@
+The **API** for this component is specified in [`RecipeList.java`](https://github.com/AY2223S2-CS2113-F13-1/tp/blob/master/src/main/java/seedu/duke/recipe/RecipeList.java)
+
+![RecipeList Component](./PlantUML/RecipeListComponent.png)
+
```
Remove the C symbols beside the Class boxes as it is not the module convention. Replace the private and public access modifiers with - and + symbols instead.

Jeraldchen commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,331 @@
+
+> The following sequence diagram shows how the help feature works:
+![Sequence Diagram for Help](./PlantUML/Help.png)
+
```
The class boxes at the bottom of your diagrams should be removed.

Jeraldchen reviewed (commented):
Overall, DG is clear and well done. There are some common syntax errors in the sequence diagrams that do not adhere to module standards as mentioned in the other comments.

Brennanzuz commented on docs/PlantUML/ArchitectureInteract.png:
I like how an example of just one command is given (`delete 1`) as it helps to understand the general flow of the program while keeping the diagram simple and readable. This is provided that `delete 1` is the most exhaustive example that shows the most classes involved.

Brennanzuz commented on docs/PlantUML/CommandComponent.puml:
```diff
@@ -0,0 +1,26 @@
+@startuml
+
```
A common problem among all groups including mine is to have the little icons. You can add:
```
hide circle
skinparam classAttributeIconSize 0
```
to remove them and follow the recommended format.

Brennanzuz commented on docs/PlantUML/Help.png:
Quotation marks around functions can be removed. I also like the mindful deletion of only the :Command object as the other classes will continue to be used in other functions, while the :Command object will no longer be used.

Brennanzuz commented on docs/PlantUML/RecipeListComponent.png:
Can the multiplicity of Step be specified? If it can, then it would be good to indicate.

Brennanzuz reviewed (commented):
Overall many many examples and code snippets to extensively show what the user can input, what is expected, and the reason for the implementation as a bonus. Ample number of diagrams to really illustrate the flow, even with a huge diagram to tie everything together. Besides understandably not adhering to the format for the class diagrams, the developer guide is very comprehensive and helpful.

KedrianLoh commented on docs/PlantUML/ArchitectureInteract.png:
The dotted line (delete 1) between UI and Duke seems a little ambiguous.

KedrianLoh commented on docs/PlantUML/Help.png:
Since the HELP message shown does not require any further user input, consider changing it to a dotted line. The UI can be shifted to the rightmost portion of the sequence diagram since it is only used after Command.

KedrianLoh reviewed (commented):
Well done overall!

liuziyang020319 committed:
Merge pull request #63 from YatPang/master

Modify UG to match actual output

jeyvia commented on src/main/java/seedu/duke/parser/Parser.java:
```diff
@@ -0,0 +1,174 @@
+        }
+        CommandType type;
+        switch (lineSpaced[0]) {
+        case "list":
```
For the switch-case, do change the magic strings in the case to constant variables, like you have done so in your `Command.java`.

jeyvia commented on src/main/java/seedu/duke/parser/Parser.java:
```diff
@@ -0,0 +1,174 @@
+        parsed.add(parsedTagStep[0].trim());
+        try {
+            parsed.add(parsedTagStep[1].trim());
+        }catch (Exception e){
```
I'm not sure how the checker did not pick up on this, but there a code-style violation here. Do check your code thoroughly for these issues :)

jeyvia commented on src/main/java/seedu/duke/parser/Parser.java:
```diff
@@ -0,0 +1,174 @@
+        }
+        return new Command(type, fullDescription);
+    }
+
+
```
Do keep the spacing between methods consistent as well! Some have 1 or 2 lines, some have no lines at all.

jeyvia commented on src/main/java/seedu/duke/parser/Parser.java:
```diff
@@ -0,0 +1,174 @@
+     * @throws IncompleteInputException if full description input is missing the description or due date or both.
+     */
+    public static ArrayList<String> parseRecipe(String description) throws IncompleteInputException {
+        ArrayList<String> parsed = new ArrayList<>();
```
You could follow more conventional naming for this variable. To my understanding these are your parsed command parameters, perhaps something a little more descriptive? :)

jeyvia commented on src/main/java/seedu/duke/ui/UI.java:
```diff
@@ -0,0 +1,160 @@
+    public void showInvalidStepMessage() {
+        System.out.println(StringLib.INVALID_STEP);
+    }
+}
```
Good job minimising the use of magic strings here! But do consider changing as many magic strings to constants as possible!

jeyvia commented on src/main/java/seedu/duke/recipe/StepList.java:
```diff
@@ -0,0 +1,85 @@
+    public ArrayList<Step> getList() {
+        return stepList;
+    }
+
```
Well done with the method naming conventions!

jeyvia commented on src/main/java/seedu/duke/parser/Parser.java:
```diff
@@ -0,0 +1,174 @@
+    private static boolean matchString(String input, String regex) {
+        String matcher = input;
+        int count = 0;
+        while (matcher.contains(regex)){
```
There are violations with regards to spacing here as well, do keep a look out for these!

Aung-Phone-Naing committed:
Strips all leading spaces and converts command words to lower case to ensure detectability

Aung-Phone-Naing committed:
Adds link tto Jar file

Aung-Phone-Naing committed:
Fixes minor gramatical error when viewing some dishes with a flavor that starts with a vowel.

liuziyang020319 committed:
Merge branch 'master' into PE-D-Fixes-Branch

Aung-Phone-Naing committed:
Fixes expected output for viewing recipe

Aung-Phone-Naing committed:
Merge branch 'PE-D-Fixes-Branch' of https://github.com/Aung-Phone-Naing/tp into PE-D-Fixes-Branch

liuziyang020319 committed:
Merge pull request #106 from Aung-Phone-Naing/PE-D-Fixes-Branch

PE-D bug fixes

YatPang committed:
Fix Issues #92, #102, #103

YatPang committed:
Fix Issues #92, #102, #103

YatPang committed:
Fix Issues #92, #97, #102, #103

liuziyang020319 committed:
Fetch add edit feature

YatPang committed:
Fix Issues #69, #92, #97, #102, #103

YatPang committed:
Fix Issues #69, #92, #97, #102, #103

YatPang committed:
Fix Issues #69, #92, #97, #102, #103

YatPang committed:
Fix Issues #69, #92, #97, #102, #103

Aung-Phone-Naing committed:
Merge pull request #107 from YatPang/master

Fix Issues #92, #102, #103

YatPang committed:
Follow-Up on #107

YatPang committed:
Follow-Up on #107

Aung-Phone-Naing committed:
Merge pull request #108 from YatPang/master

Follow-Up on #107

liuziyang020319 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-F13-1/tp

# Conflicts:
#	data/1-Hotpot.txt
#	src/main/java/seedu/duke/ui/StringLib.java

liuziyang020319 committed:
Fetch: Add edit method

YatPang committed:
Fix Issues #83

LimHongYao committed:
Merge pull request #110 from YatPang/master

Fix Issue #83

LimHongYao committed:
Fixed out of bounds handling for ingredientList
Pending decision on error format

Aung-Phone-Naing committed:
Removes the SAVE SUCCESS srtring from file writing method,
Adds view by name,
Add Junit testing for VIEW command,
Change "Find" to "View" in FindTest.

Aung-Phone-Naing committed:
Changes VIEW command instruction

Aung-Phone-Naing committed:
Merge branch 'master' into PE-D-Fixes-Branch

YatPang committed:
Add JUnit test for searchByTag

YatPang committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

Aung-Phone-Naing committed:
Resolve merge conflict

Aung-Phone-Naing committed:
Fixes merge conflict

YatPang committed:
Add JUnit test for searchByTag

LimHongYao committed:
Merge pull request #112 from YatPang/master

Add JUnit test for searchByTag

liuziyang020319 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-F13-1/tp

# Conflicts:
#	src/main/java/seedu/duke/command/Command.java
#	src/main/java/seedu/duke/command/CommandType.java
#	src/main/java/seedu/duke/recipe/RecipeList.java

liuziyang020319 committed:
Fetch: fixed coflict

liuziyang020319 committed:
Fetch: Fixed lay out bug

liuziyang020319 committed:
Fixed

YatPang committed:
Merge pull request #109 from liuziyang020319/master

Fetch: add edit method

YatPang committed:
Fix Issue #82

YatPang committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	src/main/java/seedu/duke/parser/Parser.java

YatPang committed:
Fix Issue #82

YatPang committed:
Fix Issue #82

YatPang committed:
Change Save Format

YatPang committed:
Change Save Format

Aung-Phone-Naing committed:
Fixes indentation

Aung-Phone-Naing committed:
Merge branch 'master' into PE-D-Fixes-Branch

Aung-Phone-Naing committed:
Fixes string variable name

Aung-Phone-Naing committed:
Fixes String variable classifier

Aung-Phone-Naing committed:
Fixes String declaration order

Aung-Phone-Naing committed:
Reorders variable declaration

Aung-Phone-Naing committed:
Fixes "View" to "Find"

Aung-Phone-Naing committed:
Fixes last instance of "View" to "Find"

Aung-Phone-Naing committed:
Fixes texts in Expected text to reflect actual output

YatPang committed:
Merge pull request #111 from Aung-Phone-Naing/PE-D-Fixes-Branch

Add View by name and Junit Tests

YatPang committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

Aung-Phone-Naing committed:
Merge pull request #114 from YatPang/master

Fix Issue #82

LimHongYao committed:
no message

LimHongYao committed:
Merge branch 'master' of https://github.com/LimHongYao/tp

# Conflicts:
#	src/main/java/seedu/duke/ui/StringLib.java

LimHongYao committed:
added text-ui for out of bounds edit instructions

LimHongYao committed:
Created some test cases for editIngredient
Pending solution to simulate Scanner.in

LimHongYao committed:
Modified stepbystep corrections to clear ambiguity

LimHongYao committed:
prepared object class level methods for add/remove ingredient/step methods
updated expected.txt for amended stepbystep instructions

LimHongYao committed:
Create ListEmptyException (general use)
Create getter for ingredient in IngredientList

LimHongYao committed:
modified editIngredient method for easier testing

LimHongYao committed:
add basic edit ingredient test

LimHongYao committed:
Further modification of editIngredient method to ease testing
Initial junit test for editIngredient

LimHongYao committed:
DG bold formatting issue fix

LimHongYao committed:
Create editStep junit

LimHongYao committed:
formatting fixes

LimHongYao committed:
remove extra curly

LimHongYao committed:
clear confusion between overload methods

LimHongYao committed:
Added basic editStep junit test

LimHongYao committed:
remove unused import

LimHongYao committed:
restore deleted empty editTest junit

LimHongYao committed:
Merge pull request #115 from LimHongYao/master

Fixed out of bounds for ingredientList

YatPang committed:
Add AddToRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient)

YatPang committed:
Add DeleteFromRecipe (Step/Ingredient)

Aung-Phone-Naing committed:
Adds PPP content

Aung-Phone-Naing committed:
Merge branch 'PE-D-Fixes-Branch'

Aung-Phone-Naing committed:
Updates the following:
1) Corrects link to latest JAR releases.
2) Capitalization of Headings
3) Corrects description for Parser Component and Command Component.

Aung-Phone-Naing committed:
Updates Components

Aung-Phone-Naing committed:
Updates FindRecipe plant uml diagram

Aung-Phone-Naing committed:
Updates DG format

Aung-Phone-Naing committed:
Implements Singleton for RecipeList class

Aung-Phone-Naing committed:
Fixes declaration order and style check issues

LimHongYao committed:
Merge pull request #117 from Aung-Phone-Naing/DG-Improvements

DG formatting fixes and Implements RecipeList as a Singleton

YatPang committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

Aung-Phone-Naing committed:
Fixes UG for View feature

liuziyang020319 committed:
no message

liuziyang020319 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-F13-1/tp

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient) to UG

liuziyang020319 committed:
no message

LimHongYao committed:
removed support for scrapped feature: timed step
added PPP

LimHongYao committed:
Step-by-step view now automatically terminates when the last step has been output

LimHongYao committed:
forgot to change expected.txt hehe xD

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Refactor AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

liuziyang020319 committed:
Merge pull request #118 from LimHongYao/master

Added PPP and minor view functionality edit

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	src/main/java/seedu/duke/ui/StringLib.java

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

YatPang committed:
Add AddToRecipe (Step/Ingredient), DeleteFromRecipe (Step/Ingredient)

Aung-Phone-Naing committed:
Change RecipeView and RecipeManage sequence diagrams

Aung-Phone-Naing committed:
Fixes bug in Find Recipe sequence diagram

Aung-Phone-Naing committed:
Fixes Help sequence diagram

liuziyang020319 committed:
Fetch: add exception handle in numberoverflow

liuziyang020319 committed:
Merge branch 'master' into master

Aung-Phone-Naing committed:
Fixes Help sequence diagram

liuziyang020319 committed:
Add photos

Aung-Phone-Naing committed:
Merge pull request #123 from liuziyang020319/master

Fetch: Add exception handle

liuziyang020319 committed:
Merge branch 'master' of https://github.com/liuziyang020319/tp

liuziyang020319 committed:
Format fix

YatPang committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	src/main/java/seedu/duke/ui/StringLib.java

Aung-Phone-Naing committed:
Adds sequence diagram for recipe steps edit feature

YatPang committed:
Add Flexibility to AddToRecipe command and DeleteFromRecipe command

YatPang committed:
Add Flexibility to AddToRecipe command and DeleteFromRecipe command

YatPang committed:
Add Flexibility to AddToRecipe command and DeleteFromRecipe command

YatPang committed:
Add Flexibility to AddToRecipe command and DeleteFromRecipe command

Aung-Phone-Naing committed:
Fixes error log for VIEW feature

Aung-Phone-Naing committed:
Adds constrain for ADD feature

LimHongYao committed:
Merge pull request #124 from YatPang/master

Add Flexibility to Commands AddToRecipe and DeleteFromRecipe

liuziyang020319 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-F13-1/tp

liuziyang020319 committed:
Fix couting bugs

Aung-Phone-Naing committed:
Implements error to reject numerical only input and special characters input for ADD recipe feature

Aung-Phone-Naing committed:
Updates minor language for VIEW feature

Aung-Phone-Naing committed:
Merge branch 'Constrain-Add-Recipe-Input-Branch' into DG-Improvements

liuziyang020319 committed:
no message

liuziyang020319 committed:
Fixed: checkstyle

liuziyang020319 committed:
Fixed some bugs in addtorecipe  and deletefromrecipe

liuziyang020319 committed:
Fixed: checkstyle bugs

liuziyang020319 committed:
Fixed: syntax mistake

liuziyang020319 committed:
Fix checkstyle bugs

YatPang committed:
Update Yat Pang's PPP

LimHongYao committed:
Merge pull request #129 from liuziyang020319/master

Fix bug in adding and deleting

YatPang committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

LimHongYao committed:
add hongyao's photo

Aung-Phone-Naing committed:
Fixes sequence diagrams and DG format with InvalidInputCharacterException add (missed in previous commit)

LimHongYao committed:
Merge pull request #130 from YatPang/master

Update YatPang's PPP

LimHongYao committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

Aung-Phone-Naing committed:
Fixes the deactivation of Commands in sequence diagram to extend until the "X" of the destroy call

Aung-Phone-Naing committed:
Updates DG Instructions for Manual Testing along with removal of v3.0 user stories and minor formatting changes.

Aung-Phone-Naing committed:
Merge pull request #131 from LimHongYao/master

added photo

Aung-Phone-Naing committed:
Merge branch 'DG-Improvements'

Aung-Phone-Naing committed:
Merge branch 'master' of https://github.com/Aung-Phone-Naing/tp

Merge conflicts resolved

# Conflicts:
#	src/main/java/seedu/duke/parser/Parser.java
#	src/main/java/seedu/duke/ui/StringLib.java

Aung-Phone-Naing committed:
Fixes visual bug in DG for VIEW feature

LimHongYao committed:
Merge pull request #132 from Aung-Phone-Naing/master

DG Improvements based on Javier's suggestions & Input filter for Add recipe feature

YatPang committed:
Add Yat Pang's image

Aung-Phone-Naing committed:
Merge pull request #133 from YatPang/master

Add Yat Pang's image

YatPang committed:
Update YatPang's PPP and UG

liuziyang020319 committed:
Fetch: fix some changes in UG

liuziyang020319 committed:
Merge pull request #134 from YatPang/master

Update YatPang's PPP and UG

liuziyang020319 committed:
Modify: rename the file as a correct name

liuziyang020319 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-F13-1/tp

liuziyang020319 committed:
Fetch: Add some increment of PPP

liuziyang020319 committed:
Add the most part of PPP

liuziyang020319 committed:
Update the table

liuziyang020319 committed:
Try to fix the layout

liuziyang020319 committed:
Fix another layout issue

liuziyang020319 committed:
Fix another bugs

Aung-Phone-Naing committed:
Merge pull request #135 from liuziyang020319/master

Add PPP

Aung-Phone-Naing committed:
no message

Aung-Phone-Naing committed:
Revert "no message"

This reverts commit 374f514af8f0355aa7f20a772a2b7e13570e2ee4.

Aung-Phone-Naing committed:
Merge branch 'master' of https://github.com/Aung-Phone-Naing/tp

liuziyang020319 committed:
Modify: Fix PPP layout

Aung-Phone-Naing committed:
Updates PPP and DG welcome messgage to picture

liuziyang020319 committed:
Fix bug in UG

liuziyang020319 committed:
Merge pull request #136 from Aung-Phone-Naing/master

Updates welcome message for DG to picture

LimHongYao committed:
Merge pull request #137 from liuziyang020319/master

Fix some bugs in layout

LimHongYao committed:
update README to remove duke

LimHongYao committed:
Updated PPP to match set formatting

LimHongYao committed:
Add default case/else condition warnings for code quality

LimHongYao committed:
editStep and editIngredient handling of non-number and overflow inputs

LimHongYao committed:
remove unused import

YatPang committed:
Merge pull request #138 from LimHongYao/master

Added PPP, handling for non-number and overflow inputs

Aung-Phone-Naing committed:
Updates App description for README and UG

Aung-Phone-Naing committed:
Merge branch 'master' of https://github.com/Aung-Phone-Naing/tp

# Conflicts:
#	docs/README.md

Aung-Phone-Naing committed:
Fixes UG description for deletefromrecipe feature

Aung-Phone-Naing committed:
Minor UG formatting fix

Aung-Phone-Naing committed:
Updates UG to add specification for quick start

LimHongYao committed:
Merge pull request #139 from Aung-Phone-Naing/master

UG Updates

Aung-Phone-Naing committed:
Fixes UG description for addtorecipe

Aung-Phone-Naing committed:
Fixes UG expected strings for view feature

Aung-Phone-Naing committed:
Updates UG view strings

LimHongYao committed:
fixed bug where delete to less than 1 ingredient is possible

LimHongYao committed:
fixed empty recipe list output for addtoingredient

Aung-Phone-Naing committed:
Updates help messgae to include all commands and minor UG format table update

Aung-Phone-Naing committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

Aung-Phone-Naing committed:
Fixes formatting error in StringLib

LimHongYao committed:
solved same issue for deletefromrecipe

LimHongYao committed:
req expected

LimHongYao committed:
removed unused imports

LimHongYao committed:
fix tests

YatPang committed:
Merge pull request #141 from LimHongYao/master

emptylist bug fixes

Aung-Phone-Naing committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

LimHongYao committed:
Merge pull request #140 from Aung-Phone-Naing/master

UG and Help command updates

liuziyang020319 committed:
Fix markdown bugs

liuziyang020319 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-F13-1/tp

Aung-Phone-Naing committed:
Changes name of PPP to match github username

liuziyang020319 committed:
Merge branch 'master' into master

Aung-Phone-Naing committed:
Updates DG acknowledgements

LimHongYao committed:
insert missing ingredient edit implementation

Aung-Phone-Naing committed:
Merge pull request #142 from liuziyang020319/master

Fix markdown bugs

liuziyang020319 committed:
Fix layout bugs

liuziyang020319 committed:
Merge branch 'master' of https://github.com/liuziyang020319/tp

# Conflicts:
#	docs/UserGuide.md

Aung-Phone-Naing committed:
Fixes Delete feature Number Format Exception error message

Aung-Phone-Naing committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

LimHongYao committed:
add misisng step and example usage

Aung-Phone-Naing committed:
Merge pull request #143 from LimHongYao/master

insert missing ingredient edit implementation

LimHongYao committed:
Merge pull request #144 from Aung-Phone-Naing/master

DG Updates

liuziyang020319 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-F13-1/tp

liuziyang020319 committed:
Merge remote-tracking branch 'upstream/master'

liuziyang020319 committed:
Fix grammar bugs in log

LimHongYao committed:
grammar check for recipelist

LimHongYao committed:
Merge pull request #145 from liuziyang020319/master

Fix lay out bugs in PPP

Aung-Phone-Naing committed:
Change StepsList Grammar

LimHongYao committed:
fixed grammar for add and delete commands

liuziyang020319 committed:
Merge pull request #146 from LimHongYao/master

grammar check for recipelist

Aung-Phone-Naing committed:
Fixes step delete quit message grammar

Aung-Phone-Naing committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

LimHongYao committed:
Merge pull request #147 from Aung-Phone-Naing/master

Step List Grammar update

liuziyang020319 committed:
UG grammar fixed

Aung-Phone-Naing committed:
Fixes DG bugs on github pages

LimHongYao committed:
Merge pull request #149 from Aung-Phone-Naing/master

Fixes DG bugs on github pages

Aung-Phone-Naing committed:
Merge pull request #148 from liuziyang020319/master

UG grammar fixed

Aung-Phone-Naing committed:
Final UG format fixes

Aung-Phone-Naing committed:
Final DG format fixes

Aung-Phone-Naing committed:
Merge pull request #150 from Aung-Phone-Naing/master

Final UG  and DG format fixes

LimHongYao committed:
PPP finalised

LimHongYao committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

YatPang committed:
Merge pull request #151 from LimHongYao/master

PPP finalised for hongyao

YatPang committed:
Update YatPang's PPP

Aung-Phone-Naing committed:
Merge pull request #152 from YatPang/master

Update YatPang's PPP

Aung-Phone-Naing committed:
Adds Phone's PPP

Aung-Phone-Naing committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

LimHongYao committed:
Merge pull request #153 from Aung-Phone-Naing/master

Aung-Phone-Naing committed:
Update PPP

Aung-Phone-Naing committed:
Merge branch 'AY2223S2-CS2113-F13-1:master' into master

Aung-Phone-Naing committed:
Adds more PRs to PRs reviewed for PPP

YatPang committed:
Merge pull request #154 from Aung-Phone-Naing/master

Final PPP Update
